src/pkg/p2p/preheat/provider/kraken.go

```diff
@@ -0,0 +1,131 @@
+package provider
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/docker/distribution/manifest/schema2"
+	common_models "github.com/goharbor/harbor/src/common/models"
+	"github.com/goharbor/harbor/src/common/utils"
+	"github.com/goharbor/harbor/src/pkg/p2p/preheat/dao/models"
+	"github.com/goharbor/harbor/src/pkg/p2p/preheat/provider/auth"
+	"github.com/goharbor/harbor/src/pkg/p2p/preheat/provider/client"
+	"github.com/goharbor/harbor/src/pkg/registry"
+	"time"
```

Comment:
Suggest to move `time` up to be grouped with other standard packages.

Reply:
ok

Reply:
ok